os/kernel/log_dump/log_dump.c

```diff
@@ -239,3 +419,38 @@ size_t log_dump_read(FAR char *buffer, size_t buflen)
 
 	return ret;
 }
+
+int log_dump(int argc, char *argv[])
+{
+	struct mq_attr attr;
+
+	attr.mq_maxmsg = 32;
+	attr.mq_msgsize = sizeof(struct log_dump_msg_s);
+	attr.mq_flags = 0;
+
+	/* Create log dump message queue */
+	g_logdump_mq_fd = mq_open("logdump_s", O_RDWR | O_CREAT, 0666, &attr);
+	if (g_logdump_mq_fd == (mqd_t) ERROR) {
+		lldbg("Fail to open mq, errno %d. EXIT!\n", errno);
+		return 0;
+	}
+
+	log_dump_init();
```

Comment:
This function has some failures but you don't check it and proceed next step. Is it ok?

Reply:
added failure condition